design-system/website/pages/components/fields.tsx

```diff
@@ -157,6 +176,31 @@ export default function FieldsPage() {
           On + Disabled
         </Switch>
       </FieldWrapper>
+      <h2>Date Picker</h2>
+      <FieldWrapper>
+        <BasicDatePicker />
+      </FieldWrapper>
+      <br />
```

Comment:
I'm assuming these are here to ensure the calendar appears below the input in the docs, harmless QOL improvement imo but happy to remove if necessary.